generator/standalone_main.cc

```diff
@@ -209,6 +209,9 @@ int main(int argc, char** argv) {
     for (auto const& retry_code : service.retryable_status_codes()) {
       args.emplace_back("--cpp_codegen_opt=retry_status_code=" + retry_code);
     }
+    if (service.omit_client()) {
+      args.emplace_back("--cpp_codegen_opt=omit_client=true");
```

Comment:
Opt: You may want to add a case to `generator/internal/codegen_utils_test.cc`.

Reply:
Done.